src/scenes/storeScene.js

```diff
@@ -0,0 +1,924 @@
+        this.cards.forEach(card => card.preload());
+
+        // Sprites for the whole scene (all 3 cards)
+        this.load.image('liste', "sprites/StoreScene/part1/rayon01_04_liste.png");
+        this.load.image('basket', "sprites/StoreScene/part1/rayon01_02_panier.png");
+        this.load.image('basket-front', "sprites/StoreScene/part1/rayon01_02_panier-02front.png");
+        
+        this.load.image('rature', "sprites/StoreScene/part1/rayon01_05_rature.png");
+        
+
+        this.nextCardArrow = this.load.spritesheet(
+            'next-card-arrow',
+            'sprites/StoreScene/part1/rayon01_06-rayonsuivant-spritesheet_100x100.png',
+            { frameWidth: 100, frameHeight: 100 }
+        );
+    }
+
+    addArrow() {
+        // Create arrow sprites
+        this.anims.create({
+            key: 'next-card-anim',
+            frameRate: 15,
+            frames: this.anims.generateFrameNames('next-card-arrow'),
+            repeat: -1
+        });
+
+        // Position relative to scene width
+        this.nextCardButton = this.add.sprite(
+            this.cameras.main.width * 0.39,
+            120,
+            'next-card-arrow'
+        ).play('next-card-anim');
+        
+        this.nextCardButton.depth = 4;
+
+        let scene = this;
+        this.nextCardButton.setInteractive().on('pointerdown', () => scene.nextCard() );
+    }
+
+    takeObject(object_name) {
+      // return; // debug
+      
+      console.log('take', object_name);
+      let object = this.children.getByName(object_name);
+      object.depth = 5;
+      
+      let yDistance = this.basket.y - object.y; // between 280 and 1340
+      console.log('yDistance:', yDistance)
+      let target_objects = [object]
+      if(object_name == 'pate_spaghetti01'){
+        // also move the other pack
+        target_objects.push(this.children.getByName('pate_spaghetti02'));
+      }
+      
+      // Move to basket
+      this.tweens.add({
+          targets: target_objects,
+          x: this.basket.x + (Math.random() * 50),
+          y: this.basket.y + 25 + (Math.random() * 25),
+          duration: 150 + (yDistance / 8),
+          ease: 'Quadratic',
+          yoyo: false,
+          loop: 0,
+          onComplete: () => {
+              // this.rature = this.add.image(this.listPositions[object_name], 'rature')
+              // todo: add flag
+              for (const item in this.checklist_done) {
+                if(object_name.indexOf(item) >= 0){
+                  if(this.checklist_done[item].done == false) {
+                    console.log('ajout rature', item)
+                    this.rature = this.add.image(this.checklist_done[item].position_x, this.checklist_done[item].position_y, 'rature')
+                    this.rature.depth = 20;
+                    this.checklist_done[item].done = true;
+                  } else {
+                    console.log('rature deja ajoutée')
+                  }
+                }else{
```

Comment:
again style preference but we would like to have code that generally has the same style everywhere so this should be `} else {` to follow the rest of the code's style.

Reply:
idem

Reply:
je ne sais pas comment gérer ça dans le `.editorconfig` par contre